exporters/kustomization.yaml.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Why not use `kustomize create` for this ?

Reply:
thats a good shout actually. i totally forgot about that command...we should also use that in our cre-infra makefiles

Reply:
When I tried using `kustomize create` for this I found some issues: 
 1) I couldn't redirect the output or specify the file location, it always wants to create a kustomize.yaml in the current directory
 2) there isn't any force/overwrite option, so if there are new files in the dir, you have to delete and then regenerate the file

So it was easier/more flexible to just crudely template it like this.